omero/developers/Matlab.txt

```diff
@@ -100,6 +99,27 @@ but it is not configurable from within MATLAB::
     c.enableKeepAlive(60); % Call session.keepAlive() every 60 seconds
     c.closeSession();      % Close session to end the keep-alive
 
+Working in a different group
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each session is created within a given context, defining not only the session
+user but also the session group. The session context can be retrieved using the
+administration service::
+
+    eventContext = s.getAdminService().getEventContext();
+    groupId = eventContext.groupId;
+
+Most read and write operations described below are performed in the context
+of the session group when using the default parameters. Since OMERO 5.1.4, it
+is possible to specify a different context than the session group for reading
+and writing data using the ``group`` parameter/key value in the OMERO.matlab
+functions. Retrieving objects by identifiers is also done across all groups by
```

Comment:
Not sure what the "also" is about here, did we just talk about something else that is also done across all groups by default?